control/frdata.py

```diff
@@ -354,23 +354,24 @@ def eval(self, omega, squeeze=True):
         
         squeeze: bool, optional (default=True)
             If True and sys is single input, single output (SISO), return a 
-            1D array or scalar depending on omega's length.
+            1D array or scalar the same length as omega.
 
         """
         omega_array = np.array(omega, ndmin=1) # array-like version of omega 
         if any(omega_array.imag > 0): 
             raise ValueError("FRD.eval can only accept real-valued omega")
 
         if self.ifunc is None:
-            elements = isin(self.omega, omega)
+            elements = np.isin(self.omega, omega) # binary array
             if sum(elements) < len(omega_array):
                 raise ValueError(
-                    "not all frequencies omega are in frequency list of FRD "
-                    "system. Try an interpolating FRD for additional points.")
+                    '''not all frequencies omega are in frequency list of FRD 
```

Comment:
doesn't that introduce a lot of unnecessary whitespace?

Reply:
You might be right. I'm not sure how triple quotes work when they're in a deep indent like that. Reverted to single quotes in next commit just to be sure.

Reply:
AFAIK dedent is only applied to docstrings